docs/typescript/typescript-testing.md

```diff
@@ -8,4 +8,9 @@ Visual Studio Code supports testing TypeScript through it's testing framework. Y
 
 ## Jest
 
-## Vitest
\ No newline at end of file
+## Vitest
+
+
+[Vitest](https://vitest.dev/) is an ESM-first, modern unit-test framework powered by Vite. It works out of the box for TypeScript/Node.js projects. You can learn how to get started and read about its features in [the Vitest guide](https://vitest.dev/guide/). To enhance your VS Code experience, you can install [the official VS Code extension](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer).
+
+![A screenshot of using Vitest extension inside VS Code](./images/testing/vitest.gif)
```

Comment:
```suggestion

[Vitest](https://vitest.dev/) is a modern Enterprise Service Management (ESM) unit-test framework powered by Vite. Vitest works out of the box for TypeScript or Node.js projects. To learn how to get started and about its features, check out the [Vitest guide](https://vitest.dev/guide/). You can enhance your VS Code experience by installing [the official Vitest VS Code extension](https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer).
```

The screenshot does not add value and is very noisy.

Reply:
Updated. ESM refers to ES modules